specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/ConnectToSourcePostgreSqlSyncTask.json

```diff
@@ -0,0 +1,71 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Data Migration Service Resource Provider",
```

Comment:
"Azure Database Migration Service..." - here and in other files.

Reply:
This is in other existing files as well. Let's keep it as is for this change and may be add a task to change it across all the files later.

Reply:
I thought we had already updated it for the public repo ones

Reply:
Already changed in all the files